dev/RELEASE_PROCESS.rst

```diff
@@ -3,40 +3,64 @@ Release Process
 
 This document describes the process for creating new releases.
 
-1. First, you should build wheels that you'd like to use for testing. That can
-   be done by following the `documentation for building wheels`_.
+1. **Increment the Python version:** Create a PR that increments the Python
+   package version. See `this example`_.
 
-2. **Testing:** Before a release is created, significant testing should be done.
+2. **Download the Travis-built wheels:** Once Travis has completed the tests,
+   the wheels from this commit can be downloaded from S3 to do testing, etc.
+   The URL is structured like this:
+   ``https://s3-us-west-2.amazonaws.com/ray-wheels/<hash>/ray-<version>-cp27-cp27mu-manylinux1_x86_64.whl``
```

Comment:
I'm not totally sure what to do about this, but the `cp27-cp27mu-manylinux1_x86_64.whl` string also needs to change depending on the Python version (and it's not just the 27 that changes and the `manylinux1`, but also the `mu` etc...

Reply:
Actually, let's just say, the URLS is structured like ``https://s3-us-west-2.amazonaws.com/ray-wheels/<hash>/<wheel-name>``

and maybe tell them how to figure out the wheel name by looking at https://ray.readthedocs.io/en/latest/installation.html#trying-snapshots-from-master